pkgs/by-name/ae/aegisub/package.nix

```
@@ -1,48 +1,55 @@
 {
   lib,
-  alsa-lib,
-  boost,
-  cmake,
   config,
-  darwin,
-  expat,
+  stdenv,
   fetchFromGitHub,
-  ffmpeg,
+  fetchurl,
+  fetchpatch,
+
+  meson,
+  cmake,
+  luajit,
+  ninja,
+  pkg-config,
+  which,
+  wrapGAppsHook,
+  wxGTK,
+  python3,
+  gettext,
+
+  boost,
+  expat,
+  ffmpeg_4,
```

I prefer to set this at all-packages.nix
```suggestion
  ffmpeg,
```